docs/reference/migration/migrate_7_4.asciidoc

```diff
@@ -99,14 +99,17 @@ wait for the rerouting process to completely finish you should add the
 [float]
 ==== Auto-release of read-only-allow-delete block
 
-If a node exceeds the flood-stage disk watermark then we add a block to all of
-its indices to prevent further writes as a last-ditch attempt to prevent the
-node completely exhausting its disk space. In earlier versions this block would
-remain in place until manually removed, causing confusion for users who
-currently have ample disk space and are not aware that they nearly ran out at
-some point in the past. From 7.4 onwards the block is automatically removed
-when a node drops below the high watermark again, with the expectation that the
-high watermark is some distance below the flood-stage watermark and therefore
-the disk space problem is truly resolved. This behaviour can be disabled by
-setting the system property `es.disk.auto_release_flood_stage_block` to
-`false`.
+If a node exceeds the flood-stage disk watermark then {es} adds the
+`index.blocks.read_only_allow_delete` block to all of its indices to prevent
+further writes, as a last-resort attempt to prevent the node completely
+exhausting its disk space. In earlier versions this block would remain in place
+until manually removed, causing confusion for users who currently have ample
+disk space and who are not aware that they nearly ran out at some point in the
+past. From 7.4 onwards the block is automatically removed when the node drops
+below the high watermark again, with the expectation that the high watermark is
+some distance below the flood-stage watermark and therefore the disk space
+problem is truly resolved. Since this block may be automatically removed, you
+can no longer rely on adding this block manually to prevent writes to an index.
+You should use the `index.blocks.read_only` block instead. This behaviour can
```

Comment:
NIT: I know it was there before but `behaviour` -> `behavior` because we tend to do it the 🇺🇸 way in docs? (I'm sorry)

Reply:
```
$ git annotate docs/reference/migration/migrate_7_4.asciidoc | grep behav
d73e2f9c5601	(Armin Braun	2019-07-15 10:21:54 +0200	68)If your application requires handling `+` as a single space you can return to the old behaviour by setting the system property
d73e2f9c5601	(Armin Braun	2019-07-15 10:21:54 +0200	69)`es.rest.url_plus_as_space` to `true`. Note that this behaviour is deprecated and setting this system property to `true` will cease
b47f74818634	(David Turner	2019-08-08 08:48:36 +0100	113)You should use the `index.blocks.read_only` block instead. This behaviour can
```

:trollface: